### PR TITLE
fix(restart_then_repair_node): filter out no_such_column_family error during restart

### PR DESCRIPTION
**Pull request by juliayakovlev**
Task https://trello.com/c/llRuLIOJ/2110-add-dbeventfilter-for-nosuchcolumnfamily-error
If this error happens during the first boot with the missing disk this issue is expected
and it's not an issue

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -255,11 +255,14 @@ def disrupt_stop_start_scylla_server(self):  # pylint: disable=invalid-name
...
     def disrupt_restart_then_repair_node(self):  # pylint: disable=invalid-name
         self._set_current_disruption('RestartThenRepairNode %s' % self.target_node)
-        self.target_node.restart()
-        self.log.info('Waiting scylla services to start after node restart')
-        self.target_node.wait_db_up(timeout=28800)  # 8 hours
-        self.log.info('Waiting JMX services to start after node restart')
-        self.target_node.wait_jmx_up()
+        # Task https://trello.com/c/llRuLIOJ/2110-add-dbeventfilter-for-nosuchcolumnfamily-error
+        # If this error happens during the first boot with the missing disk this issue is expected and it's not an issue
+        with DbEventsFilter(type='DatabaseLogEvent', line="Can't find a column family with UUID", node=self.target_node):
```
Comment (roydahan) [inline sdcm/nemesis.py:260]: Is this the only event we get there?
I think there are more events that we can filter there.

Comment (juliayakovlev) [reply sdcm/nemesis.py:260]: Ok. So we need to decide which events should be filtered

Comment (roydahan) [reply sdcm/nemesis.py:260]: All error events during the first boot of the cluster.

File: sdcm/nemesis.py [hunk 2]
```diff
@@ -255,11 +255,14 @@ def disrupt_stop_start_scylla_server(self):  # pylint: disable=invalid-name
...
+            self.target_node.restart()
+            self.log.info('Waiting scylla services to start after node restart')
+            self.target_node.wait_db_up(timeout=28800)  # 8 hours
```
Comment (roydahan) [inline sdcm/nemesis.py:263]: This shouldn't be part of the events filter.

Comment (juliayakovlev) [reply sdcm/nemesis.py]: Done

File: sdcm/nemesis.py (hunk 1 above)
Comment (juliayakovlev) [reply sdcm/nemesis.py:260]: We agreed to merge it as is and update list of errors when received

**Comment by juliayakovlev** [comment]
@roydahan  please, review

**Pull request merged**